src/naive/date/mod.rs

```diff
@@ -1427,7 +1432,7 @@ impl NaiveDate {
     /// specified.
     #[inline]
     pub const fn week(&self, start: Weekday) -> NaiveWeek {
-        NaiveWeek { date: *self, start }
+        NaiveWeek::new(*self, start)
```

Comment:
This should go before `new()`.

Reply:
Just to make sure: we are looking at `NaiveWeek::new` and `NaiveDate::week`. I don't think there is much I can reorder? Or I don't understand your commen.?

Reply:
If `week()` uses `new()`, it should be defined before it.

Reply:
Okay, I have been using that principle lately.

Reply:
It's still not looking the way I'd expect it to?

Reply:
It is not part of the same impl block or the same type.

Reply:
Ahh, sorry!